setup.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+#Test comment
```

Comment:
you probably didn't mean to commit this.

Reply:
Actually I did.

See, this is pull request from a fork to a non-primary branch of the origin to check the working of the Packit configuration.